staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -26,15 +26,15 @@ import (
 	"k8s.io/klog"
 )
 
-// NewDeltaFIFO returns a Store which can be used process changes to items.
+// NewDeltaFIFO returns a Queue which can be used to process changes to items.
 //
-// keyFunc is used to figure out what key an object should have. (It's
+// keyFunc is used to figure out what key an object should have. (It is
 // exposed in the returned DeltaFIFO's KeyOf() method, with bonus features.)
 //
-// 'keyLister' is expected to return a list of keys that the consumer of
-// this queue "knows about". It is used to decide which items are missing
-// when Replace() is called; 'Deleted' deltas are produced for these items.
```

Comment:
> It is used to decide which items are missing when Replace() is called; 'Deleted' deltas are produced for these items.

Is this not true anymore?

> modify the behavior of Delete, Replace, and Resync.

I think it's worth elaborating on how the behavior is modified.

Reply:
The old statement about Replace is true when `knownObjects != nil`.

I added elaboration on how the behavior is modified, in the comment on each relevant method.  I could not think of a coherent characterization of all of them, so in this central place I just made this reference to the three particular methods.